src/components/markdown.js

```diff
@@ -61,7 +73,7 @@ const Markdown = props => {
     if (props.username) {
       return (
         <>
-          {`<a href="${props.base}/${props.username}" target="blank"><img align="center" src="${props.icon}" alt="${props.username}" height="30" width="30" /></a>`}
+          {`<a href="${props.base}/${props.username}" target="blank" style="margin:0.5rem"><img align="center" src="${props.icon}" alt="${props.username}" height="30" width="30" /></a>`}
```

Comment:
can you explain the use of margin here?

Reply:
I've added the margins because without them the social icons were too close to each other and this is more inline with the markdown preview part where tailwind's `m-2` class is added. So I added the margins for consistency.